app/routes/index.js

```diff
@@ -1,18 +1,44 @@
-'use strict';
-
+"use strict";
+const util = require('util');
+global.TextEncoder = util.TextEncoder;
+global.TextDecoder = util.TextDecoder;
 var path = process.cwd();
 var contactFormMailer = require('../controllers/contactFormMailer.js');
 var galleryController = require('../controllers/galleryController.js');
-var billController = require('../controllers/billController.js');
 var userController = require('../controllers/userController.js');
-var bankController = require('../controllers/bankController.js');
-var withdrawController = require('../controllers/withdrawController.js');
-var collectedController = require('../controllers/collectedController.js');
 var unzip = require('unzip');
 var logger = require('../../logger');
+var User = require('../models/users.js');
+var Event = require('../models/events.js');
+var Team = require('../models/team.js');
 
 module.exports = function(app, fs) {
 
+	
+const multer = require('multer');
+const storage = multer.diskStorage({
+	destination: function(req, file, cb) {
+	  cb(null, '/public/uploads/');
+	},
+	filename: function(req, file, cb) {
+	  cb(null, new Date().toISOString() + file.originalname);
+	}
+  });
+  
+  const fileFilter = (req, file, cb) => {
+	// reject a file
+	if (file.mimetype === 'image/jpeg' || file.mimetype === 'image/png') {
+	  cb(null, true);
+	} else {
+	  cb(null, false);
+	}
+  };
+  
+  const upload = multer({
+	storage: storage,
+	fileFilter: fileFilter
+  });
+
```

Comment:
If this is not working please refer to the following page from node docs utilising [node-formidable](https://nodejs.org/en/knowledge/HTTP/servers/how-to-handle-multipart-form-data/). I have tried it out, and file upload works with this approach. Make sure you uninstall multer before switching to formidable.

Reply:
Done